src/eth/rpc/rpc_server.rs

```diff
@@ -15,6 +16,8 @@ use jsonrpsee::IntoSubscriptionCloseResponse;
 use jsonrpsee::PendingSubscriptionSink;
 use serde_json::Value as JsonValue;
 
+use super::rpc_internal_error;
```

Comment:
We are usually using the full name qualifier for imports.

Reply:
Sorry, auto-import